custom_components/opnsense/update.py

```
@@ -31,111 +31,100 @@ async def async_setup_entry(
-        config_entry,
-        process_entities_callback,
-        async_add_entities,
+    coordinator: OPNsenseDataUpdateCoordinator = hass.data[DOMAIN][
+        config_entry.entry_id
+    ][COORDINATOR]
+    entities: list = []
+    entity = OPNsenseFirmwareUpdatesAvailableUpdate(
+        config_entry=config_entry,
+        coordinator=coordinator,
+        entity_description=UpdateEntityDescription(
+            key=f"firmware.update_available",
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)